src/main/groovy/com/github/jrubygradle/jar/JRubyJarPlugin.groovy

```diff
@@ -116,8 +69,51 @@ class JRubyJarPlugin implements Plugin<Project> {
         project.afterEvaluate {
             project.dependencies {
                 jrubyJar group: 'org.jruby', name: 'jruby-complete', version: project.jruby.defaultVersion
+                // TODO remove hardcoded version to config
+                jrubyJar group: 'de.saumya.mojo', name: 'jruby-mains', version: '0.1.0'
+            }
+            project.tasks.withType(Jar) { task ->
+                if (task.name == 'jar') {
+                    finalizedBy( 'jrubyJar' )
```

Comment:
this finalizeBy is a hack - any suggestions on how to do it better are welcome.

but the idea is 'jar' tasks collects all resources and on the fly gnerates those .jrubydir files for this resourecs

after this 'jrubyJar' takes the first jar and all jrubyJars and packs the fat-jar